### ISSUE TITLE
CS1566  Error reading resource

### ISSUE THREAD

**Issue by EveryOtherUsernameWasAlreadyTaken** [kevingosse/windbg-extensions#3]
I cloned the repository and tried to compile, but I get the error

> CS1566  Error reading resource -- Could not find part of the path "...runner-x64.zip"

There seems to be an issue with the build order, but I could not find it. Building "WinDbgScriptRunner.x64" explicitly before building the solution fixed it.

I'm using Visual Studio 2019 (16.2.5)

**Comment by kevingosse** [comment]
Indeed, it looks like the task creating the zip file is skipped, I don't know why.
